src/Factory.php

```diff
@@ -125,8 +127,8 @@ public static function fromDefaults(array $overrideRules = []): Config
             ->in(getcwd())
             ->exclude(static::DEFAULT_EXCLUDED_DIRS)
             ->notName(static::DEFAULT_EXCLUDED_NAME)
-            ->name('*.phtml') // PhpCsFixer adds *.php and *.phpt already
-            ->name('.php_cs.dist'); // @todo: has no effect since Finder ignores hidden files
+            ->name(static::DEFAULT_INCLUDED_NAME)
+            ->ignoreDotFiles(false); // Mainly to avoid missing `.php-cs-fixer.dist.php`
```

Comment:
Rather than include all dot files, is it worth just adding this specific filename to the list of files we process?

Reply:
That's exactly what this is fixing - we were adding a specific filename, but it had no effect unless all dot files are included. I also think it's highly unlikely we'd have a bunch of hidden `.php` files that we wouldn't want to include. And even if we did, this change doesn't prevent a consumer of this package configuring around.